web/components/notifications/notification-types.tsx

```diff
@@ -0,0 +1,899 @@
+        )}
+      </>
+    </NotificationFrame>
+  )
+}
+
+function TaggedUserNotification(props: {
+  notification: Notification
+  highlighted: boolean
+  isChildOfGroup?: boolean
+}) {
+  const { notification, highlighted, isChildOfGroup } = props
+  const { sourceUserName, sourceUserUsername, sourceContractTitle } =
+    notification
+  return (
+    <NotificationFrame
+      notification={notification}
+      isChildOfGroup={isChildOfGroup}
+      highlighted={highlighted}
+      icon={
+        <AvatarNotificationIcon notification={notification} symbol={'🏷️'} />
+      }
+      link={getSourceUrl(notification)}
+    >
+      <>
+        <UserLink
+          name={sourceUserName || ''}
+          username={sourceUserUsername || ''}
+          className={'relative flex-shrink-0 hover:text-indigo-500'}
+        />{' '}
+        tagged you{' '}
+        {!isChildOfGroup && (
+          <span>
+            on <PrimaryNotificationLink text={sourceContractTitle} />
+          </span>
+        )}
+      </>
+    </NotificationFrame>
+  )
+}
+
+function FollowNotification(props: {
+  notification: Notification
+  highlighted: boolean
+  isChildOfGroup?: boolean
+}) {
+  const { notification, highlighted, isChildOfGroup } = props
+  const { sourceUserName, sourceUserUsername } = notification
+  return (
+    <NotificationFrame
+      notification={notification}
+      isChildOfGroup={isChildOfGroup}
+      highlighted={highlighted}
+      icon={
+        <AvatarNotificationIcon
+          notification={notification}
+          symbol={
+            <Col className="h-5 w-5 items-center rounded-lg bg-gradient-to-br from-gray-400 to-gray-200 text-sm">
+              ➕
+            </Col>
+          }
+        />
+      }
+      link={getSourceUrl(notification)}
+    >
+      <>
+        <UserLink
+          name={sourceUserName || ''}
+          username={sourceUserUsername || ''}
+          className={'relative flex-shrink-0 hover:text-indigo-500'}
+        />{' '}
+        followed you
+      </>
+    </NotificationFrame>
+  )
+}
+
+function LiquidityNotification(props: {
+  notification: Notification
+  highlighted: boolean
+  isChildOfGroup?: boolean
+}) {
+  const { notification, highlighted, isChildOfGroup } = props
+  const {
+    sourceUserName,
+    sourceUserUsername,
+    sourceText,
+    sourceContractTitle,
+  } = notification
+  return (
+    <NotificationFrame
+      notification={notification}
+      isChildOfGroup={isChildOfGroup}
+      highlighted={highlighted}
+      icon={
+        <AvatarNotificationIcon notification={notification} symbol={'💧'} />
+      }
+      link={getSourceUrl(notification)}
+    >
+      <>
+        <UserLink
+          name={sourceUserName || ''}
+          username={sourceUserUsername || ''}
+          className={'relative flex-shrink-0 hover:text-indigo-500'}
+        />{' '}
+        added{' '}
+        {sourceText && <span>{formatMoney(parseInt(sourceText))} of</span>}{' '}
+        liquidity{' '}
```

Comment:
Perhaps this should be subsidized? I think that's the current wording on the contract

Reply:
Yeah I don't really like the current wording, I think it's confusing and I haven't found a good way to just put it all into one sentence. It's currently 2 lines with the second line just being how much liquidity was added which seems unnecessary